regression/regression_var.sh

```diff
@@ -279,6 +283,7 @@ export gps_dtype="gps_bnd"
 export regression_vfydir="$noscrub/regression"
 
 # Define debug variable - If you want to run the debug tests, set this variable to .true.  Default is .false.
+#export debug=".true."
```

Comment:
Should this commented line be added?

Reply:
I believe it can be removed.
I added it just for convenience to switch in longer wall-time.